api/src/main/java/io/grpc/Server.java

```diff
@@ -104,8 +104,12 @@ public int getPort() {
 
   /**
    * Initiates an orderly shutdown in which preexisting calls continue but new calls are rejected.
-   * This call will not wait for preexisting calls finish before returning. After this call returns,
-   * this server has released the listening socket(s) and may be reused by another server.
+   * After this call returns, this server has released the listening socket(s) and may be reused by
+   * another server.
+   *
+   * Note that this method will not wait for preexisting calls finish before returning.
```

Comment:
preexisting calls finish => preexisting calls to finish

Reply:
Doh! Good catch